documentation/reference/qfieldcloud/projects.en.md

```diff
@@ -6,9 +6,15 @@ tx_slug: documentation_reference_qfieldcloud_projects
 # Projects
 
 Projects are the main data containers on QField and QFieldCloud.
+
 Users can create any number of projects.
+
 Projects must contain a single `.qgs`/`.qgz` QGIS file, and may in addition contain any combination of geospatial files -- GeoPackages, Shapefiles, TIFs -- or data files such as photos, PDFs etc.
-Files cannot be shared between projects, unless [localized layers](../../how-to/outside-layers.md) are used.
+
+To ensure optimal performance, it is recommended to keep the size of any file sizes below 2GB.
+Larger files than this may take significantly longer to process and lead to [jobs](jobs.md) issues.
+
+Files cannot be shared between projects, unless [shared layers](../../how-to/outside-layers.md) are used.
```

Comment:
Is shared layers the common terminology? Then I'd also change the filename (different pr)